modules/programs/nushell.nix

```
@@ -145,11 +145,24 @@ in {
     };
 
     environmentVariables = mkOption {
-      type = types.attrsOf types.str;
+      type = types.attrsOf hm.types.nushellValue;
       default = { };
-      example = { FOO = "BAR"; };
+      example = literalExpression ''
+        {
+          FOO = "BAR";
+          LIST_VALUE = [ "foo" "bar" ];
+          NU_LIB_DIRS = lib.concatStringsSep ":" [ ./scripts ];
+          PROMPT_COMMAND = lib.hm.nushell.mkNushellInline "{|| \"> \"}";
+          ENV_CONVERSIONS.PATH = {
+            from_string = lib.hm.nushell.mkNushellInline "{|s| $s | split row (char esep) }";
+            to_string = lib.hm.nushell.mkNushellInline "{|v| $v | str join (char esep) }";
+          };
+        }
+      '';
       description = ''
-        An attribute set that maps an environment variable to a shell interpreted string.
+        Environment variables to be set.
+
+        Inline values can be set with `lib.hm.gvariant.mkUint32`.
```

Is this accurate? Shouldn't this be:
```suggestion
        Inline values can be set with `lib.hm.nushell.mkNushellInline`.
```